superset/tasks/cache.py

```diff
@@ -326,5 +329,9 @@ def update_datasources_cache(cache_timeout=24 * 60 * 60):
                     force=True, cache=True, cache_timeout=cache_timeout)
                 database.all_view_names_in_database(
                     force=True, cache=True, cache_timeout=cache_timeout)
-            except Exception:
+            except Exception:  # pylint: disable=broad-except
                 logger.exception('An error occurred!')
+
+# register `update_datasources` as a Celery task; we don't use a decorator in
+# the function because it can be called directly by `superset/cli.py`
+celery_app.task(name='datasources-cache')(update_datasources)
```

Comment:
Out of curiosity: Can we be more aggressive with the cache updates? Why 24 hour instead of 5 min or something?  Fresher cache == more delightful user experience.

Reply:
@youngyjd worked on this, but I think 1 day is the balance between how often the metadata changes vs. how expensive the queries are. Maybe he has more insight here.

Also, this is now configurable when setting up the celery task, eg:

```python
CELERYBEAT_SCHEDULE = {
    'refresh-sqllab-datasources-cache-daily': {
        'task': 'datasources-cache',
        'schedule': crontab(minute=0, hour=8),
    },
}
```